content/en/blog/_posts/2023-05-04-statefulset-autodelete.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: blog
+title: 'Kubernetes 1.27: StatefulSet PVC Auto-Deletion (beta)'
+date: 04-05-2023
+slug: kubernetes-1-27-statefulset-pvc-auto-deletion-beta
+---
+
+**Author:** Matthew Cary (Google)
+
+Kubernetes v1.27 graduated to beta a new policy mechanism for
+[StatefulSets](/docs/concepts/workloads/controllers/statefulset/) that controls the lifetime of
+their [PersistentVolumeClaims](/docs/concepts/storage/persistent-volumes/) (PVCs). The new PVC
+retention policy lets users specify if the PVCs generated from the StatefulSet spec template should
+be automatically deleted or retrained when the StatefulSet is deleted or replicas in the StatefulSet
+are scaled down.
+
+## What problem does this solve?
+A StatefulSet spec can include Pod and PVC templates. When a replica is first created, the
```

Comment:
If we ever choose to add markdown linting, technically there should be a line break between the title and content lines.

Reply:
done